zerver/management/commands/delete_old_idempotentmessages.py

```diff
@@ -0,0 +1,20 @@
+from datetime import timedelta
+from typing import Any
+
+from django.utils.timezone import now as timezone_now
+from typing_extensions import override
+
+from zerver.lib.management import ZulipBaseCommand
+from zerver.models import IdempotentMessage
+
+
+class Command(ZulipBaseCommand):
```

Comment:
IMO we should make this run automatically on some schedule if we go with the table approach, otherwise this introduces a command that people running Zulip would have to run manually, which isn't a great user experience.

Alternatively, we could swap to use Redis and have the keys expire themselves, which would remove the need for this.